onnxruntime/core/framework/utils.h

```diff
@@ -236,6 +236,32 @@ int32_t ONNXTensorElementDataTypeToProtoTensorType(ONNXTensorElementDataType);
 
 std::string GetPrepackedInitializerName(const std::string& initializer_name, const std::string& node_name);
 
+size_t GetMemoryAlignedOffset(size_t current_offset);
+
+size_t CalculateTensorShapeVectorMemoryUsage(TensorShapeVector& tensor_shape_vector);
+
+Tensor ConvertPackedBufferAndShapeToTensor(onnxruntime::AllocatorPtr& alloc,
+                                           const onnxruntime::Tensor& weights,
+                                           size_t packed_weights_size_,
+                                           TensorShape weight_shape_,
```

Comment:
1) Tensor shape is passed by value? Did you mean to pass span?
2) Trailing underscores are reserved for member variables
3) I am confused what is the result of this function? Is the last parameter also an output?